Use MacPorts perl to avoid build failure with older perl on OS X 10.8
and earlier
https://sourceforge.net/p/isync/bugs/37/
--- src/drv_proxy_gen.pl.orig	2017-08-05 13:02:13.000000000 -0500
+++ src/drv_proxy_gen.pl	2017-10-04 01:15:53.000000000 -0500
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!@PREFIX@/bin/perl
 #
 # mbsync - mailbox synchronizer
 # Copyright (C) 2017 Oswald Buddenhagen <ossi@users.sf.net>
